profile.md

```diff
@@ -497,6 +497,17 @@ When groups are asserted (in an Access Token or ID Token, or both), it is a stat
 
 When a capability is asserted (only in Access Tokens), it is relative to the VO’s coarse-grained authorization; the resource only maps the token to a VO, then uses the capabilities in the token for fine-grained authorization within the VO’s authorized area.  In this way, the VO, not the resource, manages the authorizations within its area.
 
+For a storage resource, the "authorized area" is typically implemented as a set of one or more request prefixes assigned to a VO, corresponding to file paths on a filesystem.  Requests should be authorized by evaluating the scopes within the context of these prefixes.  For example, suppose a resource server `https://storage.site.org` has associated an issuer `https://vo.example.com` with prefix `/vo`.  Then, a token with scopes `storage.read:/ storage.create:/stageout` can authorize:
```

Comment:
Ciao Enrico,
would it be possible to have such "/vo-disk" and "/vo-tape" directories replaced with symlinks pointing to real directories located under a common VO root directory (e.g. "/vo") to which a token gets mapped?  Like this:

/vo-disk -> /vo/vo-disk
/vo-tape -> /vo/vo-tape

It would allow token requests to refer to "/", "/vo-disk" or "/vo-tape" as needed, all interpreted relative to the single VO area root directory.

While the VO keeps using its historic prefixes, pollution of "/vo" can be avoided by making it read-only.
OTOH, the VO might be interested in a more elaborate configuration, which would allow it to phase out its use of the prefix for the more frequently used disk area:

/vo-disk -> /vo/vo-disk -> .

In practice it might be easier to have the symlinks point in the other direction instead, though:

/vo/vo-disk -> /vo-disk
/vo/vo-tape -> /vo-tape

In either case, the layout could remain backward compatible for as long as needed, while allowing the VO to take advantage of configuration enhancements that are available with tokens, e.g. permitted by having the same namespace across SEs at different sites.

What do you think?

Reply:
Ciao @enricovianello @giacomini,
does the PR at least describe what StoRM **currently** implements?

Reply:
Yes, it does. This PR explains exactly how StoRM WebDAV currently works.

About the previous question, technically speaking the symlinks should work. We need to test this kind of deployment because it's something we were avoiding to do and, of course, I need to discuss it with CNAF T1 storage colleagues.

p.s. I apologize for this late answer.